packages/fxa-settings/src/components/Ready/index.tsx

```diff
@@ -4,14 +4,44 @@
 
 import React from 'react';
 import { RouteComponentProps } from '@reach/router';
-import { Localized } from '@fluent/react';
+import { FtlMsg } from 'fxa-react/lib/utils';
 import { ReactComponent as PulseHearts } from './account-verified.svg';
 import { logViewEvent, usePageViewEvent } from '../../lib/metrics';
 
 type ReadyProps = {
   serviceName?: string;
   continueHandler?: Function;
-  viewName: string;
+  viewName: viewNameType;
+};
+
+type viewNameType =
```

Comment:
Nit: we usually pascal case our types, e.g. `ViewNameType`. Love having this list though! 🎉

Reply:
Thank you for catching this, fixed it!!

Reply:
Borderline interesting <kbd>grep</kbd>, but here's all 432 `type` declarations (including 27 lowercase `type`s; 17 exported, 10 local):

```sh
git grep -Eoh "^(export\s)?type\s\w+" packages | sort | uniq | wc -l # 432
git grep -Eoh "^(export\s)?type\s[a-z]\w+" packages | sort | uniq | wc -l # 27
```


```sh
git grep -Eno "^(export\s)?type\s[a-z]\w+" packages | sort | uniq
```
| FILE | CODE
|:----|:----
packages/functional-tests/types.d.ts:1: | `type hexstring`
packages/fxa-admin-panel/src/lib/config.ts:95: | `type headQuerySelectorType`
packages/fxa-auth-client/lib/types.d.ts:1: | `type hexstring`
packages/fxa-auth-server/config/index.ts:2128: | `export type conf`
packages/fxa-event-broker/src/jwtset/set.interface.ts:14: | `export type deleteEvent`
packages/fxa-event-broker/src/jwtset/set.interface.ts:19: | `export type passwordEvent`
packages/fxa-event-broker/src/jwtset/set.interface.ts:25: | `export type profileEvent`
packages/fxa-event-broker/src/jwtset/set.interface.ts:31: | `export type securityEvent`
packages/fxa-event-broker/src/jwtset/set.interface.ts:41: | `export type subscriptionEvent`
packages/fxa-event-broker/src/queueworker/sqs.dto.ts:101: | `export type loginSchema`
packages/fxa-event-broker/src/queueworker/sqs.dto.ts:113: | `export type passwordSchema`
packages/fxa-event-broker/src/queueworker/sqs.dto.ts:120: | `export type profileSchema`
packages/fxa-event-broker/src/queueworker/sqs.dto.ts:128: | `export type productCapability`
packages/fxa-event-broker/src/queueworker/sqs.dto.ts:130: | `export type subscriptionUpdateSchema`
packages/fxa-event-broker/src/queueworker/sqs.dto.ts:94: | `export type deleteSchema`
packages/fxa-payments-server/src/index.tsx:112: | `type getVerifiedAccessTokenArgs`
packages/fxa-payments-server/src/lib/config.ts:115: | `type headQuerySelectorType`
packages/fxa-payments-server/src/lib/hooks.tsx:36: | `type useCheckboxStateResult`
packages/fxa-payments-server/src/lib/test-utils.tsx:143: | `type onChangeFunctionType`
packages/fxa-payments-server/src/lib/test-utils.tsx:151: | `type onBlurFunctionType`
packages/fxa-react/lib/hooks.tsx:7: | `type useBooleanStateResult`
packages/fxa-settings/src/components/Settings/DataBlock/index.tsx:16: | `type actions`
packages/fxa-settings/src/components/Settings/DataBlock/index.tsx:17: | `type actionFn`
packages/fxa-settings/src/react-app-env.d.ts:7: | `type hexstring`
packages/fxa-shared/dto/auth/payments/coupon.ts:23: | `export type couponDetailsSchema`
packages/fxa-shared/dto/auth/payments/iap-subscription.ts:24: | `export type iapExtraStripeInfoSchema`
packages/fxa-shared/dto/auth/payments/invoice.ts:120: | `export type subsequentInvoicePreview`
packages/fxa-shared/dto/auth/payments/invoice.ts:133: | `export type subsequentInvoicePreviewsSchema`
packages/fxa-shared/dto/auth/payments/invoice.ts:67: | `type line_item`
packages/fxa-shared/dto/auth/payments/invoice.ts:74: | `export type firstInvoicePreviewSchema`